src/Sarif.UnitTests/Converters/FxCopConverterTests.cs

```diff
@@ -418,7 +397,40 @@ public void FxCopConverter_CreateIssue_FakeContext_Member()
                     {"Category", "FakeCategory"},
                     {"FixCategory", "Breaking" }
                 }
-            }, FxCopConverter.CreateIssue(context));
+            };
+
+            var expectedLogicalLocationComponents = new[]
+            {
+                new LogicalLocationComponent
+                {
+                    Name = "mybinary.dll",
+                    Kind = LogicalLocationKind.ClrModule
+                },
+                new LogicalLocationComponent
+                {
+                    Name = "mynamespace",
+                    Kind = LogicalLocationKind.ClrNamespace
+                },
+                new LogicalLocationComponent
+                {
+                    Name = "mytype",
+                    Kind = LogicalLocationKind.ClrType
+                },
+                new LogicalLocationComponent
+                {
+                    Name = "mymember(string)",
+                    Kind = LogicalLocationKind.ClrFunction
+                }
+            };
+
+            var converter = new FxCopConverter();
+            Result result = converter.CreateResult(context);
+
+            result.Should().Be(expectedResult);
+
+            converter.LogicalLocationsDictionary.Keys.Should().ContainSingle(expectedLogicalLocation);
```

Comment:
As a side effect of creating the `Result`, the logical location component information has been stored in the converter's `LogicalLocationsDictionary`, which we mentioned earlier.